### PR TITLE
Add Rubocop

### PR DESCRIPTION
**Pull request by paroga**

### CODE REVIEW

File: .rubocop_todo.yml [hunk 1]
```diff
@@ -0,0 +1,991 @@
...
+  Exclude:
+    - 'plugins/current_orders/foodsoft_current_orders.gemspec'
+    - 'plugins/discourse/foodsoft_discourse.gemspec'
+    - 'plugins/documents/foodsoft_documents.gemspec'
+    - 'plugins/links/foodsoft_links.gemspec'
+    - 'plugins/messages/foodsoft_messages.gemspec'
+    - 'plugins/polls/foodsoft_polls.gemspec'
+    - 'plugins/printer/foodsoft_printer.gemspec'
+    - 'plugins/uservoice/foodsoft_uservoice.gemspec'
+    - 'plugins/wiki/foodsoft_wiki.gemspec'
```
Comment (wvengen) [inline .rubocop_todo.yml:30]: Could this become something like `plugins/*/*.gemspec` in `.rubocop.yml`?
As these plugins are bundled with Foodsoft, they'd have the same minimum version as Foodsoft.

**Review by wvengen — COMMENTED** [review]
Super!!
One comment, and the specs seem to fail (no, I didn't review _all_ changes).

File: .rubocop_todo.yml (hunk 1 above)
Comment (paroga) [reply .rubocop_todo.yml:30]: done

**Comment by paroga** [comment]
i reduced this change to whitespace only, so that it should not change any behavior

**Review by wvengen — APPROVED** [review]
:tada:

**Pull request merged**